hbase-server/src/test/java/org/apache/hadoop/hbase/replication/regionserver/TestRegionReplicaReplicationEndpointNoMaster.java

```diff
@@ -253,7 +250,8 @@ public void testRegionReplicaReplicationEndpointReplicate() throws Exception {
     when(context.getMetrics()).thenReturn(mock(MetricsSource.class));
 
     replicator.init(context);
-    replicator.start();
+    replicator.startAsync();
```

Comment:
Our ReplicationEndpoint interface extends Guava's Service interface. The Service interface has evolved. start() has been replaced by startAsync(). startAndWait has been replaced with startAsync() and awaitRunning(). stop() has been replaced by stopAsync(). stopAndWait has been replaced with stopAsync() and awaitTerminated()